style.css

```diff
@@ -0,0 +1,429 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Arial, Helvetica, sans-serif;
+    scroll-behavior: smooth;
+    color: #f4f5f5;
+  }
+
+body {
+    background-color: #030e0f;
+}
+
+.container {
+    height: 100vh;
+    display: grid;
+    grid-template: 
+    "left feed right" 1fr /
+    0.17fr 0.63fr 0.2fr;
+}
+
+.left {
+    border-right: #30635a 1px solid;
+    grid-area: left;
+    display: grid;
+    grid-template: 
+    "logo"0.1fr
+    "home"0.1fr
```

Comment:
u could have made it neater by making spaces or even use grid template coloumns/rows then grid template areas